content/en/docs/architecture/ci-operator.md

```diff
@@ -362,6 +364,44 @@ releases:
         upper: "4.5.0-0"
 {{< / highlight >}}
 
+### Testing with a Cluster from a Cluster Pool
+The `cluster_claim` below claims an `ocp` cluster with `4.7` version on `aws`. The owner of the `aws` account with which the cluster is installed is `dpp`. If the claimed cluster is running in `1h`, the environment variable `${KUBECONFIG}` stores the path to the `admin`'s `kubeconfig` file that can be used to contacting the cluster. The `kubeadmin`'s password is stored in `${KUBEADMIN_PASSWORD_FILE}`.
+
+```yaml
+- as: e2e
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    owner: dpp
+    product: ocp
+    timeout: 1h0m0s
+    version: "4.7"
+  steps:
+    test:
+    - as: claim
+      commands: |
+        printenv KUBECONFIG
+        printenv KUBEADMIN_PASSWORD_FILE
+        oc whoami
+        oc config view
+        oc get node
+      from: cli
+      resources:
+        requests:
+          cpu: 100m
+          memory: 200Mi
+```
+
+The job fails if there are no cluster pools providing such a cluster, or the time `1h` runs out waiting for the cluster to be ready. From our experience with `aws`, it takes
+
+- almost no time to claim a running cluster in the pool;
+- 3 - 6 minutes to wake up a [hibernating](https://github.com/openshift/hive/blob/master/docs/hibernating-clusters.md) cluster. A cluster is hibernating after it has not been claimed for sometime after beining provisioned;
+- 40 to 60 minutes to create a new cluster if all the pre-installed clusters in the pool are taken by other jobs.
+
+It is supported to [create a cluster pool with a cloud account provided by a CI user](/docs/how-tos/cluster-claim/). In that case, set `owner` as defined as the label of the pool. Choose `dpp` as `owner` if the team or the product that you work for does not have a dedicated pool.
+
+Note that `cluster_claim` and `cluster_profile` are mutually exclusive because [the later](/docs/architecture/step-registry/#implicit-lease-configuration-with-cluster_profile) indicates installing a cluster on demand, instead of claiming a pre-installed cluster in a pool.
```

Comment:
```suggestion
Note that `cluster_claim` and `cluster_profile` are mutually exclusive because [the latter](/docs/architecture/step-registry/#implicit-lease-configuration-with-cluster_profile) indicates installing a cluster on demand, instead of claiming a pre-installed cluster in a pool.
```

Reply:
Updated.